packages/rancher-gatekeeper/generated-changes/patch/templates/gatekeeper-audit-deployment.yaml.patch

```diff
@@ -11,12 +11,26 @@
          livenessProbe:
            httpGet:
              path: /healthz
-@@ -96,7 +96,7 @@
+@@ -96,13 +96,17 @@
            runAsUser: 1000
        hostNetwork: {{ .Values.audit.hostNetwork }}
        imagePullSecrets:
 -        {{- toYaml .Values.image.pullSecrets | nindent 8 }}
+-      nodeSelector:
+-        {{- toYaml .Values.audit.nodeSelector | nindent 8 }}
 +        {{- toYaml .Values.images.pullSecrets | nindent 8 }}
-       nodeSelector:
-         {{- toYaml .Values.audit.nodeSelector | nindent 8 }}
++      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
```

Comment:
if you supplied the nodeSelector in the original solution, it would not contain the selectors for Windows.